src/main/java/com/google/sps/servlets/BooksServlet.java

```diff
@@ -33,7 +33,7 @@
 public class BooksServlet extends HttpServlet {
 
   private final int NUM_BOOKS_PER_TERM = 5;
-  private static final String API_KEY = "API_KEY";  // Insert the API_KEY here for testing.
+  private static final String API_KEY = "AIzaSyD7NSsRElnqx6MTVSIq0-lRYe2sl2nosAk";  // Insert the API_KEY here for testing.
```

Comment:
We don't want to check in the real API KEY right?

Reply:
My bad